appveyor.yml

```diff
@@ -25,6 +25,9 @@ build:
 
 install:
 - cmd: git submodule update --init --recursive
+- cmd: echo %APPVEYOR_REPO_COMMIT%
+- cmd: echo %APPVEYOR_PULL_REQUEST_HEAD_COMMIT%
+- cmd: if not "%APPVEYOR_PULL_REQUEST_HEAD_COMMIT%" == "" if not "%APPVEYOR_PULL_REQUEST_HEAD_COMMIT%" == "%APPVEYOR_REPO_COMMIT%" git reset --soft "%APPVEYOR_PULL_REQUEST_HEAD_COMMIT%" --
```

Comment:
Could you please explain this line?
May be add a comment

Reply:
Yes, done. I should have followed my own objections against having the documentation in the PR description.
Also FormAbout and the NBug reports referred to the temporary merge commit.

Reply:
Why git-reset and not git-rev-parse "%APPVEYOR_PULL_REQUEST_HEAD_COMMIT%^1"

Reply:
Otherwise the Sha1 in the 'ThisAssembly' class populated by 'GitInfo' and so that is displayed in the application will still be at the wrong value.

Reply:
Exactly. `APPVEYOR_PULL_REQUEST_HEAD_COMMIT` already contains the commit hash of interest.
The command is the _soft_ reset which does not affect the files in the working dir at all.